src/lex/tests.rs

```diff
@@ -171,8 +171,6 @@ fn test_float_literals() {
 fn test_num_errors() {
     assert_err("1e");
     assert_err("1e.");
-    assert_err("1e100000");
-    assert_err("1e-100000");
```

Comment:
Why were these tests removed?

Reply:
These tests made sure that errors were raised when floats overflowed/underflowed.